pkg/store/store.go

```diff
@@ -661,9 +661,9 @@ func convertPromQLMatcherToFilter(matcher storepb.LabelMatcher) (opentsdb.Filter
 		if items, ok := rx.List(); ok {
```

Comment:
Will `List()` ever return `true` with a regexp wildcard in there?

Reply:
No, List doesn't handle OpStar, so can never do that. Clarified in the comment that List should be called before Wildcard though.